core/dbt/cli/params.py

```diff
@@ -112,6 +112,13 @@
     help="Stop execution on first failure.",
 )
 
+no_skip_on_failture = click.option(
+    "--no-skip-on-failture",
+    envvar="DBT_NO_SKIP_ON_FAILTURE",
+    help="If specified, dbt will proceed with downstream models even the dependent model failed.",
```

Comment:
```suggestion
    help="Proceed with downstream nodes even if an upstream node fails.",
```

This language is more in line with that of the other option help messages.